### PR TITLE
Feature/updates

### PR DESCRIPTION
**Pull request by iruzevic**
- updating component and blocks
You can run PR via commits

### CODE REVIEW

File: blocks/init/src/Blocks/custom/carousel/docs/readme.mdx [hunk 1]
```diff
@@ -13,3 +13,13 @@ wp boilerplate use_block --name=carousel
...
 We used image block as a inner child but you can use anything you like. If you want to also add image block please check the image block readme.
+
+## Important
+
+We don't provide the library used for carousel block it is up to you to implement any library you want to use in your project.
+How ever we prefere to use Swiper as our carousel of choice.
+
+To install swiper run this command in your project:
+```shell
+npm audit fix
```
Comment (dingo-d) [inline blocks/init/src/Blocks/custom/carousel/docs/readme.mdx:24]: Wrong copy paste? 😄

**Review by dingo-d — CHANGES_REQUESTED** [review]
One small comment about documentation otherwise looks good 👍🏼

File: blocks/init/src/Blocks/custom/carousel/docs/readme.mdx (hunk 1 above)
Comment (iruzevic) [reply blocks/init/src/Blocks/custom/carousel/docs/readme.mdx:24]: jap :D

Comment (iruzevic) [reply blocks/init/src/Blocks/custom/carousel/docs/readme.mdx:24]: fixed

**Review by goranalkovic-infinum — APPROVED** [review]

File: blocks/init/src/Blocks/custom/carousel/docs/readme.mdx [hunk 2]
```diff
@@ -13,3 +13,13 @@ wp boilerplate use_block --name=carousel
 ## Note
 
 We used image block as a inner child but you can use anything you like. If you want to also add image block please check the image block readme.
+
+## Important
+
+We don't provide the library used for carousel block it is up to you to implement any library you want to use in your project.
+How ever we prefere to use Swiper as our carousel of choice.
```
Comment (gabriel-glo) [inline blocks/init/src/Blocks/custom/carousel/docs/readme.mdx:20]:
```suggestion
However, we prefer to use Swiper as our carousel of choice.
```

File: scripts/components/link-toolbar-button/link-toolbar-button.js [hunk 3]
```diff
@@ -33,13 +34,10 @@ export const LinkToolbarButton = ({
...
 	};
 
-	const urlAttribute = `${componentName}Url`;
-	const opensInNewTabAttribute = `${componentName}IsNewTab`;
-
 	const unlinkButton = () => {
 		setAttributes({
-			[urlAttribute]: undefined,
-			[opensInNewTabAttribute]: undefined,
+			[`${onChangeUrl}`]: undefined,
```
Comment (gabriel-glo) [inline scripts/components/link-toolbar-button/link-toolbar-button.js:39]: Is this string literal really needed?

**Review by gabriel-glo — CHANGES_REQUESTED** [review]
Just a few comments

**Review by dingo-d — APPROVED** [review]

**Review by IvanGrginovInf — APPROVED** [review]

**Review by IvanGrginovInf — APPROVED** [review]
Apart from what @gabriel-glo commented it looks good

**Review by volfkarlo — CHANGES_REQUESTED** [review]
Just a question :) 

Great job! 😄

File: blocks/init/src/Blocks/components/button/components/button-toolbar.js [hunk 4]
```diff
@@ -42,6 +42,8 @@ export const ButtonToolbar = (attributes) => {
 							anchorRef={ref}
 							title={label}
 							textDomain={'eightshift-frontend-libs'}
+							onChangeUrl={getAttrKey('buttonUrl', attributes, manifest)}
+							onChangeIsNewTab={getAttrKey('buttonIsNewTab', attributes, manifest)}
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply blocks/init/src/Blocks/components/button/components/button-toolbar.js:46]: jes I think Goran described this in the previous comment

**Pull request merged**